.env.ci.example

```
@@ -0,0 +1,65 @@
+
+BCRYPT_ROUNDS=12
+
+LOG_CHANNEL=stack
+LOG_STACK=single
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=pgsql
+DB_HOST=postgres
+DB_PORT=5432
+DB_DATABASE=test
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_DATABASE key should go before the DB_HOST key

(UnorderedKey)